### ISSUE TITLE
✅ 🐞 Bug Report: Sticky Container loses padding when sticky is "active" after updating to 3.16

### ISSUE THREAD

**Issue by datverse** [elementor/elementor#23758]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi, in new update 3.16.0 & 3.16.1
Sticky Container Padding will be 0 when Sticky is Active.
Please check

### Steps to reproduce

1. Update to v3.16.0 / v3.16.1
2. Create new Page => Edit with Elementor
3. Create new Container (Container A) => add some Widget, example Heading Widget
4. Create new Container => add text to have height for test
5. Setting Container A to Sticky by: Advanced => Motion Effects => Sticky: Top (Issue got with Stay in Column: On/Off)
6. Save
7. View Page in Browser

=> Scroll down => Container A Sticky Top "Active" => Get Issue: Padding = 0

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt

					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.5.20
	PHP Version: 8.2.10
	PHP Memory Limit: 64M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.1
	Site URL: https://domain.com
	Home URL: https://domain.com
	WP Multisite: No
	Max Upload Size: 8 MB
	Memory limit: 1024M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Asia/Ho_Chi_Minh
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/114.0

== Active Plugins ==

	Elementor
		Version: 3.16.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.16.0
		Author: Elementor.com

	 WooCommerce
		Version: 8.1.0
		Author: Automattic

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	woocommerce: Active


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Editor Top Bar: Active
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Page Transitions: Inactive
	Notes: Inactive
	Loop: Active
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Active

== Log ==


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
				

````

</details>

**Comment by datverse** [comment]
I try to set
- "Optimized DOM Output": "Inactive"/"Active"
- "Improved Asset Loading": "Inactive"/"Active"
- "Improved CSS Loading": "Inactive"/"Active"

in Elementor/Settings/Features. But issue still have.

**Comment by perks98** [comment]
I could replicate the same issue so that when set to sticky the left and right padding disappears when active. It seems they have replaced some CSS variables so rather than using '--padding-left' it is now '--padding-inline-start', but it seems they didn't update the CSS for when it is set to sticky so it is referencing the old variables rather than the new.

![Screenshot_875](https://github.com/elementor/elementor/assets/55829113/8daa4114-276a-46d1-a917-b966de77457a)

**Comment by datverse** [comment]
> I could replicate the same issue so that when set to sticky the left and right padding disappears when active. It seems they have replaced some CSS variables so rather than using '--padding-left' it is now '--padding-inline-start', but it seems they didn't update the CSS for when it is set to sticky so it is referencing the old variables rather than the new.
> 
> ![Screenshot_875](https://user-images.githubusercontent.com/55829113/267627897-8daa4114-276a-46d1-a917-b966de77457a.jpg)

@perks98 I see the issue is ".elementor-sticky---active" don't use "CSS logical properties"

**Comment by hein-obox** [comment]
@nicholaszein 
We will need the padding styling inside the sticky styling.

**Comment by conradomd** [comment]
Workaround that I'm using:

selector .elementor-sticky--active {
    padding-left: var(--padding-block-start);
    padding-right: var(--padding-block-end);
}

**Comment by portrayaloflife** [comment]
+1

**Comment by nicholaszein** [comment]
Hey everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved with the release of Elementor Pro v3.16.1! 🥳 

✅ Feel free to check it out and update your plugins to the new versions! Elementor Core has some fixes of its own with the **3.16.3** release.

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Thank you for your time and the feedback! 🙏 

Cheers 🥂
